apps/institution-send-mail-scheduler/src/main/java/it/pagopa/selfcare/institution/entity/PecNotification.java

```diff
@@ -1,42 +1,22 @@
 package it.pagopa.selfcare.institution.entity;
 
 import io.quarkus.mongodb.panache.common.MongoEntity;
-import lombok.Getter;
-import org.bson.codecs.pojo.annotations.BsonId;
+import io.quarkus.mongodb.panache.reactive.ReactivePanacheMongoEntity;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.experimental.FieldNameConstants;
 import org.bson.types.ObjectId;
 
-@Getter
-@MongoEntity(collection="pecNotification")
-public class PecNotification {
-    @BsonId
+@EqualsAndHashCode(callSuper = true)
+@Data
+@FieldNameConstants(asEnum = true)
+@MongoEntity(collection="PecNotification")
+public class PecNotification extends ReactivePanacheMongoEntity{
+
     private ObjectId id;
```

Comment:
we are saving as [string](https://github.com/pagopa/selfcare-ms-core/blob/main/connector-api/src/main/java/it/pagopa/selfcare/mscore/model/pecnotification/PecNotification.java)

Reply:
so we need random UUID instead of ObjectId?